CHANGELOG.md

```diff
@@ -83,6 +83,15 @@ Grafana is now using Alpine 3.10 as docker base image.
 
 Please consider migrating from PhantomJS to the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
 
+# 6.3.6 (2019-09-23)
+
+### Features / Enhancements
+* **Database**: Rewrite system statistics query to perform better. [#19178](https://github.com/grafana/grafana/pull/19178), [@papagian](https://github.com/papagian)
```

Comment:
This should probably be under bug fixes. If you add the label type/bug to the PR and re-generate changelog it should pick it up. Otherwise change it manually.